pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -367,6 +369,25 @@ func configureSegmentstoreTLSSecret(podSpec *corev1.PodSpec, p *api.PravegaClust
 	}
 }
 
+func configureECSTLSSecret(podSpec *corev1.PodSpec, p *api.PravegaCluster) {
+	if p.Spec.TLS.IsSecureECS() {
+		vol := corev1.Volume{
+			Name: tlsVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				Secret: &corev1.SecretVolumeSource{
+					SecretName: p.Spec.TLS.Static.EcsSecret,
+				},
+			},
+		}
+		podSpec.Volumes = append(podSpec.Volumes, vol)
+
+		podSpec.Containers[0].VolumeMounts = append(podSpec.Containers[0].VolumeMounts, corev1.VolumeMount{
+			Name:      tlsVolumeName,
+			MountPath: tlsMountDir,
```

Comment:
Same comment as other PR: 2 secrets using the same mount path (_tlsMountDir_) won't work -  unless you use a projected volume : https://kubernetes.io/docs/tasks/configure-pod-container/configure-projected-volume-storage/

Reply:
Thanks for the valid input. Now the ECS TLS materials are put together with other Segmentstore TLS materials to simplify the process.